akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -306,15 +306,31 @@ class Http2ServerSpec extends AkkaSpec with WithInPendingUntilFixed {
     }
 
     "respect the substream state machine" should {
-      "reject other frame than HEADERS/PUSH_PROMISE in idle state with connection-level PROTOCOL_ERROR (5.1)" in pending
+      abstract class SimpleRequestResponseRoundtripSetup extends TestSetup with RequestResponseProbes
+
+      "reject other frame than HEADERS/PUSH_PROMISE in idle state with connection-level PROTOCOL_ERROR (5.1)" in new SimpleRequestResponseRoundtripSetup {
+        sendDATA(9, endStream = true, HPackSpecExamples.C41FirstRequestWithHuffman)
+        expectGOAWAY(0)
```

Comment:
well the connection closing procedure still needs to be done, so at the moment it won't close the connection, so maybe I just add pending and check if the connection is closed here after a while (which than will fail) until the connection close is fixed? I guess closing the connection is a PR by it's own. WDYT?